tests/test_orient_sync_voting.py

```diff
@@ -0,0 +1,163 @@
+import os
+import os.path
+import tempfile
+
+import numpy as np
+import pytest
+from click.testing import CliRunner
+
+from aspire.abinitio import CLOrient3D, CLSyncVoting
+from aspire.commands.orient3d import orient3d
+from aspire.source import Simulation
+from aspire.utils import (
+    Rotation,
+    get_aligned_rotations,
+    register_rotations,
+    rots_to_clmatrix,
+)
+from aspire.volume import AsymmetricVolume
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
+
+RESOLUTION = [
+    40,
+    41,
+]
+
+# `None` defaults to random offsets.
+OFFSETS = [
+    0,
+    None,
```

Comment:
~ 1min 45sec under singles. Set doubles to "expensive". 

This is the majority of the unit test time, full test takes just under 2min. I could set the random offsets to "expensive" instead of doubles if you think this is too much added time.

Reply:
yeah adding almost two minutes is a bit much for no new function.  I think we can keep one case to exercise the code and the others can become expensive. (Should be similar amount of time as before then right?)  Perhaps in the future we can make them faster, but that's another project.

I think it is great to add the coverage (thanks!), just not sure need to exercise it every pytest.

Reply:
Ok, marked `None`, ie. random offsets, to be expensive. Also, doubles are marked as expensive. The total unit test time is now ~11sec.

Reply:
Nice! Thanks.